src/main/java/org/isf/generaldata/GeneralData.java

```diff
@@ -247,6 +249,7 @@ private GeneralData(String fileProperties) {
 		PATIENTPHOTOSTORAGE = myGetProperty("PATIENTPHOTOSTORAGE", DEFAULT_PATIENTPHOTOSTORAGE);
 		SESSIONTIMEOUT = myGetProperty("SESSIONTIMEOUT", DEFAULT_SESSIONTIMEOUT);
 		PARAMSURL = myGetProperty("PARAMSURL", DEFAULT_PARAMSURL);
+		ORTHANCBASEURL = myGetProperty("", DEFAULT_ORTHANCBASEURL);
```

Comment:
Here you're not getting the prop, you're just returning the default value, what means that it will not be possible to override this setting